components/infobox/commons/infobox_scene.lua

```diff
@@ -0,0 +1,97 @@
+local Class = require('Module:Class')
+local Infobox = require('Module:Infobox')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+local Localisation = require('Module:Localisation').getLocalisation
+local Flags = require('Module:Flags')
+local Links = require('Module:Links')
+
+local getArgs = require('Module:Arguments').getArgs
+
+local Scene = Class.new()
+
+function Scene.run(frame)
+    return Scene:createInfobox(frame)
+end
+
+function Scene:createInfobox(frame)
+    local args = getArgs(frame)
+    self.frame = frame
+    self.pagename = mw.title.getCurrentTitle().text
+    self.name = args.country or args.scene or self.pagename
+
+    if args.game == nil then
+        return error('Please provide a game!')
+    end
+
+    local infobox = Infobox:create(frame, args.game)
+    local nameDisplay = Scene:nameDisplay(args)
+
+    infobox :name(nameDisplay)
+            :image(args.image)
+            :centeredCell(args.caption)
+            :header('Scene Information', true)
+            :cell('Region', Scene:createRegion(args))
+            :cell('National team', (args.nationalteam ~= nil) and ('[[' .. args.nationalteam .. ']]') or nil)
+            :cell('Key people', args['key people'])
+            :cell('Events', args.events)
```

Comment:
I think it would be much better to use `fcell` and use makeLink

Reply:
for the events?
or for the national team? (just switched that to fcell)

Reply:
Events, this is a comment about line 37